### ISSUE TITLE
How to save the classification results for each video in the testing list?

### ISSUE THREAD

**Issue by ZealACMer** [USTC-Video-Understanding/I3D_Finetune#34]
Thank you for your excellent work, my question is how to save .h5 or .npz classfication score files as shown in https://github.com/rohitgirdhar/ActionVLAD/tree/master/data/hmdb51/final_logits or https://github.com/youjiangxu/seqvlad-pytorch/tree/master/logits/hmdb51, thank you so much for you kind help, looking forward to your reply, thanks a lot.
